Various technical notes/transition_infos.json

```diff
@@ -960,20 +927,96 @@
           "area_id": "0x62548B77",
           "area_name": "White Valley",
           "requires": null
-        },
-        {
-          "area_id": "0xDEDA69BC",
-          "area_name": "Jungle Canyon",
-          "requires": [
-            [
-              "GLITCHES"
-            ],
-            [
-              "pickaxe"
-            ]
-          ]
         }
       ]
     }
+  ],
+  "Spirit Fights": [
+    {
+      "area_id": "0x02C7B675",
+      "area_name": "Monkey Temple (Spirit)",
+      "default_entrance": "0xF3B4DC8E",
+      "exits": []
+    },
+    {
+      "area_id": "0x0305DC42",
+      "area_name": "Scorpion Temple (Spirit)",
+      "default_entrance": "0x4B08BBEB",
+      "exits": []
+    },
+    {
+      "area_id": "0x1F237F32",
+      "area_name": "Penguin Temple (Spirit)",
+      "default_entrance": "0x1B11EC74",
+      "exits": []
+    }
+  ],
+  "El Dorado": [
+    {
+      "area_id": "0x99885996",
+      "area_name": "Ruins of El Dorado (Jaguar)",
+      "default_entrance": "0x93F89D45",
+      "exits": []
+    },
+    {
+      "area_id": "0xCFD2FE10",
+      "area_name": "Ancient El Dorado (Pusca)",
+      "default_entrance": "0x99885996",
+      "exits": []
+    }
+  ],
+  "Native Games": [
+    {
+      "area_id": "0x0A1F2526",
+      "area_name": "Whack-A-Tuco",
+      "default_entrance": "0x05AA726C",
+      "exits": []
+    },
+    {
+      "area_id": "0x0D72E13F",
+      "area_name": "Tuco Shoot",
+      "default_entrance": "0x05AA726C",
+      "exits": []
+    },
+    {
+      "area_id": "0x9316749C",
+      "area_name": "Raft Bowling",
+      "default_entrance": "0x05AA726C",
+      "exits": []
+    },
+    {
+      "area_id": "0x7A75D1A9",
+      "area_name": "Pickaxe Race",
+      "default_entrance": "0x05AA726C",
+      "exits": []
+    },
+    {
+      "area_id": "0xE411440A",
+      "area_name": "KaBOOM!",
+      "default_entrance": "0x05AA726C",
+      "exits": []
+    }
+  ],
+  "Cutscenes": [
+    {
+      "area_id": "0x53257119",
+      "area_name": "Plane Crash (CUTSCENE)",
+      "default_entrance": "0x99885996",
+      "exits": []
+    },
+    {
+      "area_id": "0xE8362F5F",
+      "area_name": "Viracocha Monoliths (CUTSCENE)",
+      "default_entrance": "0x7652BAFC",
+      "exits": []
+    }
+  ],
+  "Unused": [
+    {
+      "area_id": "0x47508760",
+      "area_name": "Beta Volcano Crash Site",
+      "default_entrance": "0x53257119",
+      "exits": []
+    }
```

Comment:
See my above comment about major areas.

Beta volcano is also not a pre-existing connection.

Even as far as exits and requirements go, it has all the exact same as the final crash site. So swapping the volcano in for the real crash site is as simple as an ID swap.

If you want to *add* it to the pool w/o replacing, then it goes in the jungle major area

Reply:
Well you can see **my** above comment about major areas too ;)

About Beta Volcano: I have big plans for this one, so I have good reasons to included it the way I did.
This shouldn't impact anything in a negative way though, so no need to start panicking.

Reply:
Marking as resolved because we discussed this over Discord.